hs_app_netCDF/apps.py

```diff
@@ -3,6 +3,3 @@
 
 class NetCDFAppConfig(AppConfig):
```

Comment:
What is stopping us from removing the whole file?  The migrations?  Seems like after the other resource types are removed we could create a new db dump and remove the applications completely.

Reply:
That is what I thought as well. After all resource types are removed with migrations taken care of, another PR can be created to remove all the removed resource type apps as well.

Reply:
Yeah we need this file to run the new migrations for this app. Once the migration is applied we should be able to remove the app completely.